docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -40,18 +42,23 @@ from files on a filesystem.
 
 For ops, each op output can have its own IO manager, or multiple op outputs can share an IO manager. The IO manager that's used for handling a particular op output is automatically used for loading it in downstream ops.
 
-<Image
-alt="two-io-managers"
-src="/images/concepts/two-io-managers.png"
-width={3200}
-height={1040}
-/>
+Consider the following diagram. In this example, a job has two IO managers, each of which is shared across a few inputs and outputs:
 
-<!-- https://excalidraw.com/#json=4546514944786432,52cpcHHoWfzOMro2eFC6MQ -->
+<center>
+  <Image src="/images/concepts/two-io-managers.png" width={643} height={382} />
+</center>
 
-This diagram shows a job with two IO managers, each of which is shared across a few inputs and outputs.
+For assets, each asset can have its own IO manager. In the [multi-asset](/concepts/assets/multi-assets) case where multiple assets are outputted, each outputted asset can be handled with a different IO manager:
 
-For assets, each asset can have its own IO manager. In the [multi-asset](/concepts/assets/multi-assets) case where multiple assets are outputted, each outputted asset can be handled with a different IO manager.
+<center>
+  <Image
```

Comment:
This is the new diagram, which I took a shot at after reading through some of the docs. Is it correct? What could be better or corrected, if anything?